### PR TITLE
build your first dapp

### PR DESCRIPTION
**Pull request by ywy2090**
change the example to asset management.
https://test-test.readthedocs.io/en/release-2.0/docs/tutorial/sdk_application.html

### CODE REVIEW

File: docs/tutorial/sdk_application.md [hunk 1]
```diff
@@ -1,200 +1,282 @@
...
-|name  |subject   |score  |
-+======+==========+=======+
-|Alice |Math      |98     |
-+------+----------+-------+
-|Alice |Chinese   |90     |
-+------+----------+-------+
-|Bob   |English   |95     |
-+------+----------+-------+
++-----------------+------------------+
+|  asset_account  |   asset_amount   |
```
Comment (qyan-dev) [inline docs/tutorial/sdk_application.md]: account

Comment (qyan-dev) [inline docs/tutorial/sdk_application.md]: asset_value

File: docs/tutorial/sdk_application.md [hunk 2]
```diff
@@ -1,200 +1,282 @@
...
-    // 创建学生成绩表
-    function createTable() public {
+    function createTable() private {
         TableFactory tf = TableFactory(0x1001); 
-        // 参数：表名，主键字段，普通字段(使用逗号分隔)
-        tf.createTable("t_student_score", "name", "subject,score");
+        // 资产管理表, key : asset_account, field : asset_amount
+        // |  资产账户(主键)      |     资产金额       |
+        // |-------------------- |-------------------|
+        // |  asset_account      |  asset_amount     |
```
Comment (qyan-dev) [inline docs/tutorial/sdk_application.md]: account

Comment (qyan-dev) [inline docs/tutorial/sdk_application.md]: asset_value

File: docs/tutorial/sdk_application.md [hunk 3]
```diff
@@ -1,200 +1,282 @@
...
+- 查询资产金额。
+- 能够在链上进行资产转移。
 
 ## 合约开发
 
-**业务表设计：** 实现学生成绩管理，首先需要设计一个存储学生成绩的表`t_student_score`，该表字段如下：
-- name：主键，学生姓名(字符串类型)
-- subject：课程名称(字符串类型)
-- score：分数(整型)
+**业务表设计：** 实现资产管理，首先需要设计一个存储资产管理的表`t_asset_management`，该表字段如下：
```
Comment (qyan-dev) [inline docs/tutorial/sdk_application.md]: Use t_asset to be consistent with its file name Asset.sol

File: docs/tutorial/sdk_application.md [hunk 4]
```diff
@@ -254,299 +335,99 @@ compile ('org.fisco-bcos：web3sdk：2.0.2')
...
 编译项目。
 ```bash
 # 切换到项目目录
-$ cd ~/fisco/student-score-app
+$ cd ~/asset-app
 # 编译项目
 $ ./gradlew build
 ```
-编译成功之后，将在项目根目录下生成`dist`目录。dist目录下有一个`run.sh`脚本，简化项目运行。现在开始一一验证本文开始定下的需求。
+编译成功之后，将在项目根目录下生成`dist`目录。dist目录下有一个`asset_run.sh`脚本，简化项目运行。现在开始一一验证本文开始定下的需求。
```
Comment (qyan-dev) [inline docs/tutorial/sdk_application.md]: run_asset_app.sh

Comment (ywy2090) [reply docs/tutorial/sdk_application.md]: ok

**Pull request merged**